omemo.html

```diff
@@ -23,18 +23,28 @@ <h2>OMEMO Encryption</h2>
             <h3>Contents</h3>
 
             <ul>
-                <li><a href="#installing-omemo">Building with OMEMO support</a>
-                <li><a href="#prelim-setup">Setup</a>
+                <li><a href="#state">Current state</a></li>
+                <li><a href="#installing-omemo">Building with OMEMO support</a></li>
+                <li><a href="#prelim-setup">Setup</a></li>
                 <ul>
                     <li><a href="#private-key-gen">Generating Crypto Materials</a>,
                         <a href="#fingerprints">Fingerprint authentication</a>,
                         <a href="#private-conversation">Starting a private conversation</a></li>
                 </ul>
-                <li><a href="#policy">Setting OMEMO policy</a>
-                <li><a href="#ui-opts">User Interface options</a>
-                <li><a href="#omemo-message-logging">OMEMO message logging</a>
+                <li><a href="#policy">Setting OMEMO policy</a></li>
+                <li><a href="#ui-opts">User Interface options</a></li>
+                <li><a href="#omemo-message-logging">OMEMO message logging</a></li>
             </ul>
         </section>
+        <section>
+            <a name="state"></a>
+            <h3>Current state</h3>
+            <p>OMEMO support is currently only available if you build from git master.</p>
+            <p>You need to enable carbons (<code>/carbons on</code>) to have a good OMEMO experience. Otherwise you don't see messages sent from other devices.</p>
```

Comment:
I think carbon is more recommended than mandatory. Nor it's not specific to OMEMO neither it will break OMEMO if it's not enabled.

Maybe something like `Enabling carbon is recommended in order to enhance OMEMO and overall chat experience.`

Reply:
changed :)